examples/application_commands/autocompleted_command.py

```diff
@@ -1,9 +1,10 @@
 import nextcord
 from nextcord import Interaction, SlashOption
+from nextcord.ext import commands
 
 TESTING_GUILD_ID = 123456789  # Replace with your testing guild id
 
-client = nextcord.Client()
+bot = commands.Bot(command_prefix='!')
```

Comment:
In most other examples, the prefix used is `$`

```suggestion
bot = commands.Bot(command_prefix='$')
```

Reply:
When #101 is merged, the prefix can even be left out entirely